website/static/js/profile.js

```diff
@@ -647,12 +664,33 @@ var ListViewModel = function(ContentModel, urls, modes) {
 ListViewModel.prototype = Object.create(BaseViewModel.prototype);
 
 ListViewModel.prototype.addContent = function() {
-    this.contents.push(new this.ContentModel(this));
+    if (!this.institutionObjectsEmpty() && this.isValid()) {
+        this.contents.push(new this.ContentModel(this));
+    }
+    else {
+        this.changeMessage(
+            'Institution Field is Required',
+            'text-danger',
+            5000
+        );
+    }
 };
 
 ListViewModel.prototype.removeContent = function(content) {
     var idx = this.contents().indexOf(content);
-    this.contents.splice(idx, 1);
+    //  If there is more then one model, then delete it.  If there is only one, then delete it and add another.
+    if (this.contentsLength() > 1) {
+        this.contents.splice(idx, 1);
```

Comment:
This happens in both the `if` and `else` branch, so it can be pulled out of here; then you'll only need a single `if` branch (no need for `else`).